specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/nova.json

```diff
@@ -0,0 +1,1143 @@
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Deletion will complete asynchronously."
+          },
+          "204": {
+            "description": "No such cluster to delete."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "CassandraDataCenter_CreateUpdate",
+        "description": "Create or update a managed Cassandra data center. When updating, overwrite all properties. To update only some properties, use PATCH.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraDataCenterCreate": {
+            "$ref": "./examples/CosmosDBManagedCassandraDataCenterCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            },
+            "description": "Parameters specifying the managed Cassandra data center."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Accepted. The data center will be created asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            }
+          },
+          "200": {
+            "description": "Accepted. The data center will be updated asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
```

Comment:
200 should not have a response with it. 

refer: [https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc015-put-apis-that-only-return-200](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc015-put-apis-that-only-return-200)

Reply:
It should have a response. Again, see https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource, which says that 200, 201, etc. should all return at least the properties that were posted. 200 just means no-op.